Libraries/LibWeb/HTML/HTMLOutputElement.cpp

```diff
@@ -121,4 +121,11 @@ bool HTMLOutputElement::will_validate()
     return false;
 }
 
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#dom-cva-validationmessage
+String HTMLOutputElement::validation_message() const
+{
+    // Returns the empty string, as output elements are not subject to constraint validation.
+    return {};
+}
```

Comment:
I don't think this method is necessary because `FormAssociatedElement::validation_message()` would also return the empty string.